Accelerometer/CS/Package.appxmanifest

```diff
@@ -0,0 +1,50 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  IgnorableNamespaces="uap mp">
+
+  <Identity
+    Name="6b970cdc-1e41-44a0-b4c5-af6e175e6bfe"
+    Publisher="CN=Accelerometer"

```

Comment:
Why was Publisher changed from "CN=MSFT" to "CN=Accelerometer"?  This should be changed back and the line above changed from Name="6b970cdc-1e41-44a0-b4c5-af6e175e6bfe" to Name="Accelerometer" to make it consistent with the other C# applications such as hello blinky.

Reply:
Identity name and Identity Publisher name is changed as required, and well make it consistent in all the other samples as well